src/controllers/languages.js

```
@@ -2,10 +2,18 @@ const Language = require('~/models/languages')
 const createNotFoundError = require('~/utils/errorsHelper')
 const languageService = require('~/services/languages')
 
+const DEFAULT_PAGE = 1
+const DEFAULT_LIMIT = 6
+
 const getLanguages = async (req, res) => {
   try {
-    const languages = await languageService.getLanguages()
-    res.status(200).json(languages)
+    const page = parseInt(req.query.page) || DEFAULT_PAGE
+    const limit = parseInt(req.query.limit) || DEFAULT_LIMIT
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)